front/components/assistant/AssistantDetails.tsx

```diff
@@ -495,7 +505,7 @@ function DataSourceViewSelectedNodes({
         <Tree.Item
           key={node.internalId}
           label={node.titleWithParentsContext ?? node.title}
-          type={node.expandable ? "node" : "leaf"}
+          type={node.expandable && viewType !== "tables" ? "node" : "leaf"}
```

Comment:
You should be able use `canBeExpanded` (maybe).

Reply:
not really, it actually does the opposite of what's expected here